frontend/src/components/stats/ActiveNearAppsUsersByDate.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from "react";
+import ReactEcharts from "echarts-for-react";
+import * as echarts from "echarts";
+import { useTranslation } from "react-i18next";
+
+import { AccountsByDate } from "../../libraries/explorer-wamp/stats";
+
+import { Props } from "./TransactionsByDate";
+
+const ActiveNearAppsUsersByDate = ({ chartStyle }: Props) => {
+  const { t } = useTranslation();
+  const [activeAccountsByWeek, setWeekAccounts] = useState(Array());
+  const [week, setWeek] = useState(Array());
+
+  useEffect(() => {
+    // TODO(sparkyrider): Put a proper implementation in near-analytics after User Activity NEP lands and get implemented on indexer-for-explorer side.
+    // Meanwhile, @sparkyrider will manually submit the new values weekly in PRs to NEAR Explorer.
+    // XXX: Hardcoded values were generated by this query:
+    // select to_timestamp(receipt_included_in_block_timestamp / 1000000000)::date as date
+    //  ,count(*) as accountscount
+    //  from action_receipt_actions
+    // where receipt_receiver_account_id = 'v1.nearapps.near'
+    //   and to_timestamp(receipt_included_in_block_timestamp / 1000000000) > NOW() - interval '60 days'
+    // group by to_timestamp(receipt_included_in_block_timestamp / 1000000000)::Date;
+    let accounts = [
+      {
+        date: "2022-01-02 23:59:59",
+        accountsCount: 214,
+      },
+      {
+        date: "2021-01-12 23:59:59",
+        accountsCount: 277,
+      },
+      {
+        date: "2021-12-19 23:59:59",
+        accountsCount: 7902,
+      },
+      {
+        date: "2021-12-26 23:59:59",
+        accountsCount: 220,
+      },
+      {
+        date: "2022-01-02 23:59:59",
+        accountsCount: 214,
+      },
+      {
+        date: "2022-01-09 23:59:59",
+        accountsCount: 204774,
+      },
+      {
+        date: "2022-01-16 23:59:59",
+        accountsCount: 393725,
+      },
+      {
+        date: "2022-01-23 23:59:59",
+        accountsCount: 30946,
+      },
+    ];
+    if (accounts) {
```

Comment:
This condition is excessive at the moment

Reply:
Thanks for catching it! The code was based on a copy-pasted version of another component and I did not clean it up properly since I expected we would only have it in the staging preview, but things got changed and the new plan is to merge it, so indeed it needs some clean up now.